src/stories/packages/components/Select.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { useArgs } from '@storybook/preview-api';
+import type { Meta, StoryFn } from '@storybook/react';
+import { Select } from '@terra-ui-packages/components';
+import React from 'react';
+
+const meta: Meta<typeof Select> = {
+  title: 'Packages/Components/Select',
+  component: Select,
+  parameters: {
+    layout: 'centered',
+    docs: {
+      story: {
+        height: '200px',
+      },
+    },
+  },
+  tags: ['autodocs'],
+  args: { options: ['First', 'Second', 'Third'], value: 'First' },
+  argTypes: {
+    value: {
+      control: 'select',
+      description: 'a member of the options parameter',
+      options: ['First', 'Second', 'Third'],
+    },
+    isClearable: {
+      control: 'boolean',
+      description: 'whether the select can be cleared',
```

Comment:
It would be good to document the default values for all of these, like https://github.com/DataBiosphere/terra-ui/blob/cd7f5bd6d9e70e1d6ea92cd232f134f5666dc43b/src/stories/src/components/NumberInput.stories.tsx#L45